public/app/features/dashboard-scene/panel-edit/PanelDataPane/PanelDataTransformations/PanelDataTransformationsTab.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { DragDropContext, Droppable } from 'react-beautiful-dnd';
+
+import { IconName } from '@grafana/data';
+import { SceneObjectBase, SceneComponentProps, SceneDataTransformer } from '@grafana/scenes';
+import { Container, CustomScrollbar } from '@grafana/ui';
+import { TransformationOperationRows } from 'app/features/dashboard/components/TransformationsEditor/TransformationOperationRows';
+import { TransformationsEditorTransformation } from 'app/features/dashboard/components/TransformationsEditor/types';
+
+import { VizPanelManager } from '../../VizPanelManager';
+import { PanelDataPaneTabState, PanelDataPaneTab } from '../types';
+
+import { EmptyTransformationsMessage } from './EmptyTransformationsMessage';
+
+interface PanelDataTransformationsTabState extends PanelDataPaneTabState {}
+
+export class PanelDataTransformationsTab
+  extends SceneObjectBase<PanelDataTransformationsTabState>
+  implements PanelDataPaneTab
+{
+  static Component = PanelDataTransformationsTabRendered;
+  tabId = 'transformations';
+  icon: IconName = 'process';
+  private _panelManager: VizPanelManager;
+
+  getTabLabel() {
+    return 'Transformations';
+  }
+
+  getItemsCount() {
+    return null;
+  }
+
+  constructor(panelManager: VizPanelManager) {
+    super({});
+
+    this._panelManager = panelManager;
+  }
+
+  get panelManager() {
+    return this._panelManager;
+  }
+}
+
+interface TransformationEditorProps {
+  sceneDataTransformer: SceneDataTransformer;
+}
+
+function TransformationsEditor(props: TransformationEditorProps) {
+  const dataState = props.sceneDataTransformer.useState();
+  const transformationEditorRows: TransformationsEditorTransformation[] = [];
+
+  let i = 0; // Ids need to be unique for drag to change order to work
+  for (const t of dataState.transformations) {
+    if ('id' in t) {
+      transformationEditorRows.push({ id: `${i} - ${t.id}`, transformation: t });
+      i++;
+    }
+  }
```

Comment:
```suggestion
  const transformations = dataState.transformations.map(( transformation, index) => ({ id: `${index}`,  transformation})  
```

Reply:
I wrote it this way to have the type guard discern between `DataTransformerConfig` and `CustomTransformOperator`. From my understanding when editing dashboard scenes we would never really see `CustomTransformOperator` in the transformations array. I could explore other ways to get around this type issue.